assets/js/app/Data.js

```diff
@@ -9,7 +9,13 @@ LocusZoom.DataSources = function() {
     this.sources = {};
 };
 
+LocusZoom.KnownDataSources = [];
```

Comment:
Currently with all other singletons there's no distinction between the singleton members LocusZoom defines in the core, the members you as a user define, and the members you're actually using. From the perspective of Data Layers, Transformations, Scale Functions, etc. I'm not sure such a distinction really matters... there's just one collection that you can add to (or delete from at your own risk) and use whatever you need to whenever you need to. This simplifies the storage of such members as there's only one place they live, and affords future flexibility to begin (or stop) using members of any singleton well after initialization (e.g. as triggered by a UI element or something).